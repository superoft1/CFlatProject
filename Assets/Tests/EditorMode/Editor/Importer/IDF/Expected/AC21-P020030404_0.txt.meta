fileFormatVersion: 2
guid: 408b9a602ccfa4d4f85bd23529bcc004
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
