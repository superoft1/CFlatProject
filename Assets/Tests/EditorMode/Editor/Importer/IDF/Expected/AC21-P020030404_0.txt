Elbow90, P02A-000M3, (-1.0, 0.0, 0.0), (-11945.6, 9261.7, 104.7), False
(-11945.6, 9262.1, 104.7)
(-11945.2, 9261.7, 104.7)
FreeVertex: 1
Pipe, P01A-0016U, (1.0, 0.0, 0.0), (-11944.8, 9261.7, 104.7), False
(-11945.2, 9261.7, 104.7)
(-11944.3, 9261.7, 104.7)
FreeVertex: 0
Elbow90, P02A-000M3, (0.0, 0.0, 1.0), (-11943.9, 9261.7, 104.7), False
(-11944.3, 9261.7, 104.7)
(-11943.9, 9261.7, 104.4)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, 0.0, -1.0), (-11943.9, 9261.7, 104.2), False
(-11943.9, 9261.7, 104.4)
(-11943.9, 9261.7, 104.0)
FreeVertex: 0
Tee, P21A-00259, (0.0, 0.0, -1.0), (-11943.9, 9261.7, 103.7), True
(-11943.9, 9261.7, 103.5)
(-11943.9, 9261.7, 104.0)
(-11943.7, 9261.7, 103.7)
FreeVertex: 0
Pipe, P01A-0016U, (1.0, 0.0, 0.0), (-11941.5, 9261.7, 103.7), False
(-11943.7, 9261.7, 103.7)
(-11939.3, 9261.7, 103.7)
FreeVertex: 0
Elbow90, P02A-000M3, (0.0, 0.0, 1.0), (-11938.9, 9261.7, 103.7), False
(-11939.3, 9261.7, 103.7)
(-11938.9, 9261.7, 103.4)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, 0.0, -1.0), (-11938.9, 9261.7, 102.4), False
(-11938.9, 9261.7, 103.4)
(-11938.9, 9261.7, 101.4)
FreeVertex: 0
WeldOlet, P01A-0016U, (-1.0, 0.0, 0.0), (-11938.9, 9261.7, 101.4), False
(-11938.9, 9261.7, 101.4)
(-11938.9, 9261.7, 101.4)
(-11939.1, 9261.7, 101.4)
FreeVertex: 0
Pipe, P01A-000XV, (-1.0, 0.0, 0.0), (-11939.1, 9261.7, 101.4), False
(-11939.1, 9261.7, 101.4)
(-11939.2, 9261.7, 101.4)
FreeVertex: 0
Pipe, P01A-000XV, (-1.0, 0.0, 0.0), (-11939.2, 9261.7, 101.4), False
(-11939.2, 9261.7, 101.4)
(-11939.3, 9261.7, 101.4)
FreeVertex: 0
Tee, P24A-006WZ, (-1.0, 0.0, 0.0), (-11939.3, 9261.7, 101.4), True
(-11939.4, 9261.7, 101.4)
(-11939.3, 9261.7, 101.4)
(-11939.3, 9261.6, 101.4)
FreeVertex: 0
WeldNeckFlange, P61B-0032U, (0.0, -1.0, 0.0), (-11939.3, 9261.6, 101.4), False
(-11939.3, 9261.6, 101.4)
(-11939.3, 9261.6, 101.4)
FreeVertex: 1
Pipe, P01A-000XV, (-1.0, 0.0, 0.0), (-11939.4, 9261.7, 101.4), False
(-11939.4, 9261.7, 101.4)
(-11939.5, 9261.7, 101.4)
FreeVertex: 0
Elbow90, P02A-000UJ, (0.0, 1.0, 0.0), (-11939.5, 9261.7, 101.4), False
(-11939.5, 9261.7, 101.4)
(-11939.5, 9261.6, 101.4)
FreeVertex: 0
WeldNeckFlange, P61B-0032S, (0.0, -1.0, 0.0), (-11939.5, 9261.6, 101.4), False
(-11939.5, 9261.6, 101.4)
(-11939.5, 9261.6, 101.4)
FreeVertex: 1
Pipe, P01A-0016U, (0.0, 0.0, -1.0), (-11938.9, 9261.7, 101.4), False
(-11938.9, 9261.7, 101.4)
(-11938.9, 9261.7, 101.3)
FreeVertex: 0
Elbow90, P02A-000M3, (0.0, -1.0, 0.0), (-11938.9, 9261.7, 100.9), False
(-11938.9, 9261.7, 101.3)
(-11938.9, 9262.1, 100.9)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, 1.0, 0.0), (-11938.9, 9262.2, 100.9), False
(-11938.9, 9262.1, 100.9)
(-11938.9, 9262.3, 100.9)
FreeVertex: 0
WeldOlet, P01A-0016U, (0.0, 0.0, -1.0), (-11938.9, 9262.3, 100.9), False
(-11938.9, 9262.3, 100.9)
(-11938.9, 9262.3, 100.9)
(-11938.9, 9262.3, 100.8)
FreeVertex: 0
GateValve, P81B-002WG, (0.0, 0.0, -1.0), (-11938.9, 9262.3, 100.6), False
(-11938.9, 9262.3, 100.5)
(-11938.9, 9262.3, 100.8)
FreeVertex: 0
Cap, P29B-0000C, (0.0, 0.0, -1.0), (-11938.9, 9262.3, 100.5), False
(-11938.9, 9262.3, 100.5)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, 1.0, 0.0), (-11938.9, 9262.4, 100.9), False
(-11938.9, 9262.3, 100.9)
(-11938.9, 9262.6, 100.9)
FreeVertex: 0
WeldNeckFlange, P61B-003LX, (0.0, 1.0, 0.0), (-11938.9, 9262.6, 100.9), False
(-11938.9, 9262.7, 100.9)
(-11938.9, 9262.6, 100.9)
FreeVertex: 1
Pipe, P01A-0016U, (0.0, 0.0, -1.0), (-11943.9, 9261.7, 102.4), False
(-11943.9, 9261.7, 103.5)
(-11943.9, 9261.7, 101.3)
FreeVertex: 0
Elbow90, P02A-000M3, (0.0, -1.0, 0.0), (-11943.9, 9261.7, 100.9), False
(-11943.9, 9261.7, 101.3)
(-11943.9, 9262.1, 100.9)
FreeVertex: 0
Pipe, P01A-0016U, (0.0, 1.0, 0.0), (-11943.9, 9262.2, 100.9), False
(-11943.9, 9262.1, 100.9)
(-11943.9, 9262.4, 100.9)
FreeVertex: 0
WeldOlet, P01A-0016U, (0.0, 0.0, -1.0), (-11943.9, 9262.4, 100.9), False
(-11943.9, 9262.4, 100.9)
(-11943.9, 9262.4, 100.9)
(-11943.9, 9262.4, 100.8)
FreeVertex: 0
Pipe, P01A-000XV, (0.0, 0.0, -1.0), (-11943.9, 9262.4, 100.7), False
(-11943.9, 9262.4, 100.8)
(-11943.9, 9262.4, 100.7)
FreeVertex: 0
Pipe, P01A-000XV, (0.0, 0.0, -1.0), (-11943.9, 9262.4, 100.7), False
(-11943.9, 9262.4, 100.7)
(-11943.9, 9262.4, 100.7)
FreeVertex: 0
Elbow90, P02A-000UJ, (1.0, 0.0, 0.0), (-11943.9, 9262.4, 100.6), False
(-11943.9, 9262.4, 100.7)
(-11944.0, 9262.4, 100.6)
FreeVertex: 0
Pipe, P01A-000XV, (-1.0, 0.0, 0.0), (-11944.0, 9262.4, 100.6), False
(-11944.0, 9262.4, 100.6)
(-11944.1, 9262.4, 100.6)
FreeVertex: 0
WeldNeckFlange, P61B-0032S, (-1.0, 0.0, 0.0), (-11944.1, 9262.4, 100.6), False
(-11944.1, 9262.4, 100.6)
(-11944.1, 9262.4, 100.6)
FreeVertex: 1
Pipe, P01A-0016U, (0.0, 1.0, 0.0), (-11943.9, 9262.5, 100.9), False
(-11943.9, 9262.4, 100.9)
(-11943.9, 9262.6, 100.9)
FreeVertex: 0
WeldNeckFlange, P61B-003LX, (0.0, 1.0, 0.0), (-11943.9, 9262.6, 100.9), False
(-11943.9, 9262.7, 100.9)
(-11943.9, 9262.6, 100.9)
FreeVertex: 1
FreeVertexTotal: 6
